configs/permissions/inventory.json

```diff
@@ -0,0 +1,3 @@
+{
+    "*": ["*", "read", "write"]
```

Comment:
To clarify, you'll want the following permissions available for custom role creation, but you'll only create two roles, one for admin and one for viewer, but won't be using the write permission explicitly out of the gate, correct?
```
inventory:*:*
inventory:*:read
inventory:*:write
```

Reply:
Yes, I think that's what the team agreed on at this point. I'll update this PR if things change.